godot_project/editor/rendering/reference_shape_renderer/reference_shape_shader.gdshader

```diff
@@ -10,6 +14,7 @@ render_mode cull_back, unshaded;
 
 instance uniform float selected : hint_range(0.0, 1.0) = 0.0;
 instance uniform float hovered : hint_range(0.0, 1.0) = 0.0;
+instance uniform float preview: hint_range(0,1) = 1.0;
```

Comment:
are you sure this should be 1.0 by default? 🤔

Reply:
Actually, it looks like dead code, should remove it

Reply:
yep, the pr was already a second iteration on this :eyes: